www/docs/usage/commandline.md

```diff
@@ -17,47 +17,74 @@ The most recently selected versions are presented at the top of the dropdown.
 ## See all versions including beta, alpha and release candidates(rc)
 <img src="https://s3.us-east-2.amazonaws.com/kepler-images/warrensbox/tfswitch/tfswitch-v5.gif" alt="drawing" style="width: 600px;"/>
 
-1. Display all versions including beta, alpha and release candidates(rc). 
+1. Display all versions including beta, alpha and release candidates(rc).
 2. For example, `tfswitch -l` or `tfswitch --list-all` to see all versions.
 3. Hit **Enter** to select the desired version.
 
 ## Use environment variables
-You can also set the `TF_VERSION` environment variable to your desired terraform version. 
-For example:   
+
+You can also set the `TF_VERSION` environment variable to your desired terraform version.
+For example:
```

Comment:
I'd glue this together (or put two trailing periods otherwise):
```suggestion
You can also set the `TF_VERSION` environment variable to your desired terraform version. For example:
```

Reply:
Fixing